docs.md

```diff
@@ -1,6 +1,37 @@
 ---
-layout: docs-nav
+layout: default-nav
 title: Documentation
 description: mlpack | fast, flexible C++ machine learning library
-order: 2
+order: 3
 ---
+<div class="page-title-header">Documentation</div>
```

Comment:
Maybe should we just use the H1 class for this here?

```
# Documentation
```

plus the right CSS for `h1` that's currently in `page-title-header`?  It might simplify things a bit.

Reply:
Same comment as before, the class is a rich CSS - not just font size.